de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/pporder/api/impl/HUPPOrderQtyDAO.java

```diff
@@ -104,6 +104,7 @@ public boolean isHuIdIssued(@NonNull final HuId huId)
 		return Services.get(IQueryBL.class).createQueryBuilder(I_PP_Order_Qty.class)
 				.addOnlyActiveRecordsFilter()
 				.addEqualsFilter(I_PP_Order_Qty.COLUMN_M_HU_ID, huId)
+				.addEqualsFilter(I_PP_Order_Qty.COLUMN_PP_Order_BOMLine_ID, null)
```

Comment:
This filter contradicts the next line, AFAIU therefor this method will never return true.
Can you please outline why the filter is needed?

Reply:
In the next line you are applying a filter (i.e., **.addNotNull(I_PP_Order_Qty.COLUMN_PP_Order_BOMLine_ID)**) which picks records with only null values in "PP_ORDER_BOMLINE_ID" column of "PP_ORDER_QTY" table(we actually didn't understood why records corresponding to null values are only being picked here). Because of this filter By-Products produced during the Manufacturing process were not getting displayed in the HU-Bin. So to overcome that we applied this filter.